[2024-05-05T10:06:47.306+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T10:06:47.327+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample.get_files manual__2024-05-05T10:06:43.455582+00:00 [queued]>
[2024-05-05T10:06:47.336+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample.get_files manual__2024-05-05T10:06:43.455582+00:00 [queued]>
[2024-05-05T10:06:47.337+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T10:06:47.354+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-05-05 10:06:43.455582+00:00
[2024-05-05T10:06:47.364+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=79559) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-05T10:06:47.365+0000] {standard_task_runner.py:63} INFO - Started process 79561 to run task
[2024-05-05T10:06:47.364+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sample', 'get_files', 'manual__2024-05-05T10:06:43.455582+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp3s9bumfr']
[2024-05-05T10:06:47.366+0000] {standard_task_runner.py:91} INFO - Job 191: Subtask get_files
[2024-05-05T10:06:47.405+0000] {task_command.py:426} INFO - Running <TaskInstance: sample.get_files manual__2024-05-05T10:06:43.455582+00:00 [running]> on host 5bc5768c5236
[2024-05-05T10:06:47.500+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sample' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T10:06:43.455582+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-05T10:06:43.455582+00:00'
[2024-05-05T10:06:47.501+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T10:06:47.751+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T10:06:51.438+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1061 InsecureRequestWarning: Unverified HTTPS request is being made to host 'opendata.onde.go.th'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
[2024-05-05T10:06:51.559+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-05T10:06:51.560+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T10:06:51.578+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=sample, task_id=get_files, execution_date=20240505T100643, start_date=20240505T100647, end_date=20240505T100651
[2024-05-05T10:06:51.642+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T10:06:51.666+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-05T10:06:51.672+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
